src/logging.cpp

```diff
@@ -168,7 +174,7 @@ std::vector<CLogCategoryActive> ListActiveLogCategories()
     return ret;
 }
 
-std::string BCLog::Logger::LogTimestampStr(const std::string &str)
+std::string BCLog::Logger::LogTimestampStr(std::string str)
```

Comment:
I don't think you need to change the signature here. Any reason to change from a const reference?

Reply:
See https://github.com/bitcoin/bitcoin/pull/13168#discussion_r195141701